lib/pavilion/test_config/resolver.py

```diff
@@ -393,6 +397,14 @@ def load_raw_configs(self, tests, host, modes):
                         "Test suite '{}' raised a type error, but that "
                         "should never happen. {}".format(test_suite_path, err))
 
+                # Check test compatibility with current pavilion version
+                comp_error = self.check_suite_version_compatibility(test_suite_cfg)
+
+                if comp_error:
+                    raise TestConfigError(
+                        "Test suite '{}' has incompatibility issues: {}"
+                        .format(test_suite, comp_error))
+
```

Comment:
That's exactly why I did it this way, user will see all incompatibility issues at once so they only need to go and edit the test suite once. Instead of only seeing the firs occurrence, changing, and seeing another issue. I can change it, but feel like it's better to display all issues at once...